build/vue-html-to-paper.js

```diff
@@ -20,12 +20,14 @@
         let defaultName = '_blank', 
             defaultSpecs = ['fullscreen=yes','titlebar=yes', 'scrollbars=yes'],
             defaultReplace = true,
-            defaultStyles = [];
+            defaultStyles = [],
+            defaultTimeout = 10000;
```

Comment:
Set the default to 1000 only. If the user wants to have a higher timeout they can then set a higher value.